pages/scholars/_id.vue

```diff
@@ -1,13 +1,300 @@
 <template>
   <app-main-container>
     <template v-slot:title>{{ $route.params.id }}</template>
-
-    User Detail
+      <v-row>
+        <v-col
+            cols="12"
+            md="2"
+            sm="2"
+        >
+            <v-card
+              min-height="70"
+              
+            >
+              <v-card-title>
+                <v-row
+                  class="pt-5 px-5"
+                  align="center"
+                  justify="center"
+                >
+              
+                  <v-img
+                      src="https://assets.coingecko.com/coins/images/10366/large/SLP.png?1578640057"
```

Comment:
The same string is repeated 5 times
`https://assets.coingecko.com/coins/images/10366/large/SLP.png?1578640057`

Refactor and move this inside `state` from `store/index.js` file, move it in a property called `assets` and have a key name of `slp`
```js
// index.js

export const state = () => ({
  // ...
  assets: {
   slp: 'https://assets.coingecko.com/coins/images/10366/large/SLP.png?1578640057'
  }
})
```
Then from the view/page that requires to call this asset, call it by `$store.state.assets?.slp`